web/src/lib/components/admin-page/settings/storage-template/storage-template-settings.svelte

```diff
@@ -151,7 +152,7 @@
         <SupportedVariablesPanel />
       </section>
 
-      <div class="mt-4 flex flex-col">
+      <div class="flex flex-col mt-4">
```

Comment:
My linter did this, but I can un-do if you wish

Reply:
This is fine, no worries